Makefile.am

```diff
@@ -261,5 +261,5 @@ cpplint: Makefile.am
 if FOUND_CPPLINT
 	cpplint --filter=$(CPP_LINT_FILTER) $(CPP_LINT_FILES)
 else
-  $(error cpplint not found. Install forked cpplint (e.g. via pip for the latest version) and re-run configure.)
+	$(error cpplint not found. Install the forked cpplint (e.g. via pip for the latest version) and re-run configure.)
```

Comment:
Wow I would never have found that!

Reply:
TBH configure on GitHub actions which I guess is newer gave an even more ominous message (or it's just the fact its a clean checkout each time).

It was pretty blatantly obviously different when I went to edit the file on GitHub so I gave it a go and the rest is history.